### PR TITLE
Storybook infra

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/4479250922

### CODE REVIEW

File: utils/colors-vars-map.ts [hunk 1]
```diff
@@ -0,0 +1,264 @@
+export const colorsMap = [
```
Comment (shlomitc) [inline utils/colors-vars-map.ts:1]: are you sure this entire file is needed?

File: storybook/stand-alone-documentation/welcome/welcome.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Contributors
+
+monday.com’s designers, developers, and writers provide teams with universal assets—elements, components, patterns, and code—and the guidance on how to design and build with them.
+We welcome contributions to monday.com design system!
+
+### Meet the Vibe team
+
+<Contributors />
+<OtherContributorsList />
```
Comment (shlomitc) [inline storybook/stand-alone-documentation/welcome/welcome.stories.mdx:24]: I think you can remove it for now. It's a lot of code and it's a duplication of the vibe page which means double maintenance

File: package.json [hunk 3]
```diff
@@ -47,25 +49,45 @@
     "url": "https://github.com/mondaycom/monday-ui-style/issues"
   },
   "homepage": "https://github.com/mondaycom/monday-ui-style#readme",
+  "dependencies": {
+    "postcss": "8.4.4",
+    "postcss-scss": "^4.0.3",
+    "postcss-value-parser": "4.1.0"
```
Comment (shlomitc) [inline package.json:55]: what aren't they devdeps?

**Review by shlomitc — CHANGES_REQUESTED** [review]
In general - let's try in the future to review smaller PRs as it's hard to go over both the infrastructure and the content.
Usually, when not purifying the exact changes, a lot of code can be added which is not relevant.

If it's not too much effort I'd prefer to use only typescript instead of keep using JSX

File: package.json (hunk 3 above)
Comment (SergeyRoyt) [reply package.json]: Moved it

File: utils/colors-vars-map.ts (hunk 1 above)
Comment (SergeyRoyt) [reply utils/colors-vars-map.ts:1]: Removed unrelevant parts

File: storybook/stand-alone-documentation/welcome/welcome.stories.mdx (hunk 2 above)
Comment (SergeyRoyt) [reply storybook/stand-alone-documentation/welcome/welcome.stories.mdx:24]: I'd like to leave it as it is
- it just looks better with Welcome page
- It gives more consistent experience regarding Vibe storybook
- We can later fill it with more useful information
- Double maintenance will only be needed when will replace team pictures, the rest of the content is different
-  I've already put some efforts to do it

Comment (shlomitc) [reply storybook/stand-alone-documentation/welcome/welcome.stories.mdx:24]: it's just that there's so much duplication between this and the other storybooks so it's terrible for maintenance.

**Comment by SergeyRoyt** [comment]
> In general - let's try in the future to review smaller PRs as it's hard to go over both the infrastructure and the content. Usually, when not purifying the exact changes, a lot of code can be added which is not relevant.
> 
> If it's not too much effort I'd prefer to use only typescript instead of keep using JSX

I agree, let's try to separate the PRs next time on the task layer
Let's create a task and put it into backlog - I think it's a really low priority task and I don't want to place it in the same scope - the scope is already too wide here

File: .storybook/manager-head.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
+<link href="https://fonts.googleapis.com/css?family=Roboto:100,200,300,400,500&display=swap" rel="stylesheet" />
+<meta property="og:description" content="Monday UI style by monday.com" />
+<meta property="og:title" content="Monday UI style" />
+<meta property="og:image" content="https://irp.cdn-website.com/33477119/dms3rep/multi/og-banner.jpeg" />
+<title>Monday UI style</title>
+<script src="https://use.fontawesome.com/660238b999.js"></script>
```
Comment (shlomitc) [inline .storybook/manager-head.html:6]: is this needed?

File: .storybook/manager-head.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  ) {
+    storybookConfig.resizerNav.x = DEFAULT_MINIMUM_LEFT_PANEL_WIDTH;
+    localStorage.setItem("storybook-layout", JSON.stringify(storybookConfig));
+    document.location.reload();
+  } else if (storybookConfig === null) {
+    storybookConfig = { resizerNav: { x: DEFAULT_MINIMUM_LEFT_PANEL_WIDTH, y: 0 } };
+    localStorage.setItem("storybook-layout", JSON.stringify(storybookConfig));
+    document.location.reload();
+  }
+</script>
```
Comment (shlomitc) [inline .storybook/manager-head.html:63]: is this necessary?

File: .storybook/manager.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import { addons } from "@storybook/manager-api";
+import theme from "./theme";
+
+// window.STORYBOOK_GA_ID = "UA-308574295"; // TODO set STORYBOOK_GA_ID? What's is that for?
```
Comment (shlomitc) [inline .storybook/manager.js:4]: google analytics. let's remove it

File: .storybook/preview-head.html [hunk 7]
```diff
@@ -0,0 +1,103 @@
+<link rel="preconnect" href="https://fonts.googleapis.com" />
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```
Comment (shlomitc) [inline .storybook/preview-head.html:2]: is this really needed?

File: .storybook/preview-head.html [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+/>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;700&display=swap" rel="stylesheet" />
+<link
+  href="https://fonts.googleapis.com/css2?family=Figtree:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+  rel="stylesheet"
+/>
+<link
+  href="https://fonts.googleapis.com/css2?family=Figtree:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+  rel="stylesheet"
+/>
```
Comment (shlomitc) [inline .storybook/preview-head.html:16]: can these be merged into a single link?

File: .storybook/preview-head.html [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+<link href="//fonts.googleapis.com/css2?family=Noto+Kufi+Arabic:wght@300;400;500;700&display=swap" rel="stylesheet" />
+<link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@300;400;500;700" rel="stylesheet" />
+<link href="https://fonts.cdnfonts.com/css/sofia-pro" rel="stylesheet" />
+<script src="https://use.fontawesome.com/660238b999.js"></script>
```
Comment (shlomitc) [inline .storybook/preview-head.html:20]: I don't think they're necessary at this moment

**Review by shlomitc — APPROVED** [review]
just minor stuff that probably can be removed

File: .storybook/preview-head.html (hunk 7 above)
Comment (SergeyRoyt) [reply .storybook/preview-head.html:2]: Removed fonts.gstatic.com

File: .storybook/manager-head.html (hunk 5 above)
Comment (SergeyRoyt) [reply .storybook/manager-head.html]: Removed, not working

**Pull request merged**